src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+  justify-content: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  & > :nth-child(-n + 4) {
+    background-image: url("../images/star-active.svg");
+  }
```

Comment:
Add a possibility to change a modifier

Reply:
Still not fixed from the previous review